controllers/user.js

```diff
@@ -696,18 +696,21 @@ function logout(req, res) {
 async function getRemovalEnrollPage(req, res) {
   const user = req.user;
 
-  if (checkIfRemovalEnrollmentEnded(user) && !req.query.show) {
+  if (
+    checkIfRemovalEnrollmentEnded(user) &&
+    !FormUtils.canShowViaParams(req.query?.show)
```

Comment:
Q: Should we call `FormUtils.canShowViaParams(req.query?.show)` once and be caching the result?
Looks like we might be calling it multiple times in some methods (a total of 14&times; in this PR, but most of them seem to be in different methods though).

But the `getRemovalEnrollPage(req, res)` method here seems to call it 3&times;.

Reply:
Well - it checks each page's `show` URL parameter, so that can't be cached - the other piece of this is just a lookup of `AppConstants.NODE_ENV` so that is essentially required at the top of the file and thus essentially cached.

Reply:
Yeah, I think only 3 methods called it multiple times. Probably not worth over-optimizing and you can ignore.

Reply:
@pdehaan  -Ah - sorry - I missed that we are calling it multiple times in the same function, sorry. Will implement your suggestion!